### PR TITLE
Improved tab inside modal

### PR DESCRIPTION
**Pull request by thangdinh**
+ Made modal-dialog as a fall-back pseudo element when there is no
sub-element on focus
+ Fixed the problem with wrong tabbing behavior when holding mouse down
It gives essentially a much better UX when dealing with some 3rd party component such as bootstrap date-picker on the modal. 
@azirbel @igillis 
CSS: @hanakimis

### CODE REVIEW

**Comment by thangdinh** [comment]
bump. @azirbel or @igillis , could you please help review it? Gonna be short. Thanks.
@hanakimis: can you have a look at 3 lines css :)?

File: src/css/modal.less [hunk 1]
```diff
@@ -12,6 +12,13 @@
   overflow-y: auto;
 }
 
+// modal-dialog will be a fall-back focusing element where there is no
+// sub-element is on focus. We remove the outline for not making it look
+// awkward when it is on focus
```
Comment (azirbel) [inline src/css/modal.less]: "on focus" -> "in focus", here and on the line above

**Comment by azirbel** [comment]
LGTM

**Comment by chrnguyen** [comment]
I know I'm not @hanakimis , but css LGTM

**Comment by thangdinh** [comment]
Merged

**Pull request closed without merging**